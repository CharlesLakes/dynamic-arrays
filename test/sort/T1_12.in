10
41 100 40 37 95 74 53 81 29 71