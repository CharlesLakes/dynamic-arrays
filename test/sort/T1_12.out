29 37 40 41 53 71 74 81 95 100